File: src/main/cpp/kattis/fizzbuzz.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
    int x, y, n, m = 1; scanf("%d %d %d", &x, &y, &n);
    while (n--) {
        if (!(m % x) && !(m % y))   printf("FizzBuzz\n");
        else if (!(m % x))          printf("Fizz\n");
        else if (!(m % y))          printf("Buzz\n");
        else                        printf("%d\n", m);
        m++;
    }
    return 0;
}

File: src/main/java/com/algorithms/halim/cap1/FizzBuzz.java
package com.algorithms.halim.cap1;

import java.util.Scanner;

public class FizzBuzz {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x, y, n;
        x = sc.nextInt();
        y = sc.nextInt();
        n = sc.nextInt();
        StringBuilder sb = new StringBuilder();
        for (int i = 1; i <= n;i++) {
            if (i % x == 0 && i % y == 0) {
                sb.append("FizzBuzz\n");
            } else if (i % x == 0) {
                sb.append("Fizz\n");
            } else if (i % y == 0) {
                sb.append("Buzz\n");
            } else {
                sb.append(i);
                sb.append("\n");
            }
        }
        System.out.print(sb);
    }
}
